site/pages/our-work/case-study/ft/index.js

```diff
@@ -30,7 +30,7 @@ const FinancialTimesCaseStudy = ({ contactUsURL }: CaseStudyFinancialTimesProps)
     <div className={styles.header}>
       <div className={styles.header__container}>
         <div className={styles.header__imageContainer}>
-          <img src={headerImage} alt="shopping cart" className={styles.header__image} />
+          <img src={headerImage} alt="meeting" className={styles.header__image} />
```

Comment:
Could we make this alt text a bit more descriptive, please? Maybe change to "image of an Agile stand up in progress"